tools/revng-lift/BinaryFile.cpp

```diff
@@ -957,10 +963,13 @@ void BinaryFile::parseELF(object::ObjectFile *TheBinary,
 
   for (Elf_Phdr &ProgramHeader : *ProgHeaders) {
     switch (ProgramHeader.p_type) {
+    case PT_PAGEBUSTER:
     case ELF::PT_LOAD: {
+      bool pagebuster { ProgramHeader.p_type == PT_PAGEBUSTER };
```

Comment:
Just use regular assignments unless there's a reason to.
```suggestion
      bool IsPageBuster = (ProgramHeader.p_type == PT_PAGEBUSTER);
```